Rearraging of the PDHD PDS simulation worflow and updating protodune radiological generators

jroto opened the pull request:
Changes:
- I removed the Rn generator from the generators, since this applies only underground.
- I removed the 1GeV tag from the cosmic generator, since there are no beam generator inside, which is confusing.
- I updated the g4 fhicl to optimize the simulation workflow. It is simpler now, and it only remains as it was when using the seminanalytical model. The G4 fhicl for computable graph is also added.

lpaulucc reviewed (commented):
Hi José, this looks good but we should have the semi-ana model in the standard fhicl instead of the optical library. Could you change that?

lpaulucc commented:
José, one other thing is that there needs to be checked is if the Optical Backtracker is receiving both acitve and external volumes information. I think Shuaixiang identified this was not the case.

jroto commented:
- Hi @lpaulucc, I renamed the semianalytical model simulation to standard as requested.
- The issue Shuaixiang reported about the opdetbacktracker affects just the service, not the creation of the opdetbacktrackerrecords. This is already solved for the computable graph and the photon library with this PR, since the external volume dependency is removed. It would need to be solved in dunecore for the semianalytical model only.
- I've also updated the radiological generators to use the last one used in the far detectors (using decay0). Do you suggest to add some additional background source?
For example I see some Co60 in the FD APAs, and 40K in the CPAs.

lpaulucc reviewed (approved):
Thank you, José! This looks good. As a suggestion for making maintenance easier, you could create a single gen file (e.g. protodunevd_gen.fcl) where you put in all producers and then adapting each specific fhicl to inherit from this one so you don't need to change a handful of files every time you change a background or something.
Now, regarding backgrounds, I would not include any PDS related ones because the PDs have not being assayed yet. For APA, in the FD we have two models, one if the geometry represents the APA in a more realistic way and a hacked one, so not knowing exactly what we have in PDHD makes it difficult to evaluate what we could use. In VD, there is a CRP model. Maybe this should be brought up at a DRA meeting.

jroto commented:
This PR should not be merged until https://github.com/DUNE/duneopdet/pull/65 is merged.

lpaulucc commented on fcl/protodunehd/gen/prod_beam_cosmics_1GeV_protodunehd.fcl:
```diff
@@ -3,7 +3,7 @@
 #include "corsika_protodune.fcl"
 #include "protodunebeam.fcl"
-#include "dune_radiological_model.fcl"
+#include "dune_radiological_model_decay0.fcl"
```
Change to dune_radiological_model_decay0_hd_common.fcl

jroto replied:
What is the difference between one file and the other? they look very very similar.

jroto replied:
What about the VD ones? should they change too?

lpaulucc replied:
The second file is the updated one, although I cannot say if anything changes for the LAr radiologicals. In any case, I would reference the latest one, specially if people decide to include other HD specific background sources. For VD, if LAr is the only radiologicals so far, them it should be fine to keep this one. Otherwise there is the vd specific version of this file.
